libraries/pbrlib/genglsl/lib/mx_microfacet_specular.glsl

```diff
@@ -441,9 +441,9 @@ vec2 mx_latlong_projection(vec3 dir)
     return vec2(longitude, latitude);
 }
 
-vec3 mx_latlong_map_lookup(vec3 dir, mat4 transform, float lod, sampler2D sampler)
+vec3 mx_latlong_map_lookup(vec3 dir, mat4 transform, float lod, sampler2D tex_sampler)
```

Comment:
`sampler` is a reserved word in Vulkan GLSL
https://www.khronos.org/registry/OpenGL/specs/gl/GLSLangSpec.4.60.html#keywords